pennylane/configuration.py

```diff
@@ -95,28 +91,9 @@
 you can also specify settings on a device-by-device basis, by placing the options under the
 ``[plugin.device]`` settings.
```

Comment:
It looks like this info is not showing up in the docs at all

Reply:
Yes, that is true. The docstring is rather technical, so I thought it is for the "interested user who digs through the code". I suggest adding a section to "Quick reference" which explains how to use configurations. I admit, I have no clue how to do this myself, is there a template one has to change?

Reply:
Yep - sounds good. I suppose this module docstring can be copied ad-hoc over to a ReST file. Something @AroosaIjaz pointed out on Friday is that some things are a bit hidden in quick references - things that might be good to have 'top-level' are Operations (and maybe config). But this is open for discussion.